script.js

```diff
@@ -65,3 +65,18 @@ function updateUncheckedTodoCount() {
   const todosLeftElement = document.getElementById("todos-left");
   todosLeftElement.textContent = `Todos left unchecked: ${uncheckedTodos}`;
 }
+
+function setFilter(filterValue) {
+  const filterRadioButton = document.getElementById(
```

Comment:
NIT: Why do you need so complicated expression for id? You could easily select by value:
```
const filters = document.getElementById("filters");
...
function setFilter(filterValue) {
const filterRadioButton = filters.querySelector('input[value=${filterValue}]')
...
```